modules/pam_limits/pam_limits.c

```diff
@@ -832,6 +832,23 @@ set_if_null(char **dest, char *def)
     return 1;
 }
 
+static char *
+trim(char *s)
+{
+    char *p;
+
+    while (*s == ' ' || *s == '\t')
```

Comment:
Since `s` is a value returned by `strtok_r`, it can be `NULL`.

Reply:
Thanks!